modules/persistence/src/services/entries/index.ts

```diff
@@ -13,7 +14,7 @@ const entriesFromZip = async (zip) => {
     .map((entry) => deserialize(entry.getData()));
 };
 
-export const insertEntries = async (buildId, zip) => {
+export const insertEntries = async (buildId: ObjectId, zip) => {
```

Comment:
minor nitpick. not required, but there is a type library for zip objects: https://www.npmjs.com/package/@types/adm-zip